inst/@sym/repmat.m

```diff
@@ -51,18 +52,20 @@
     print_usage ();
   end
 
-  cmd = { '(A, n, m) = _ins'
-          'if n == 0 or m == 0:'
-          '    return sp.Matrix(n, m, [])'
-          'if A is None or not A.is_Matrix:'
-          '    A = sp.Matrix([A])'
-          'L = [A]*m'
-          'B = sp.Matrix.hstack(*L)'
-          'L = [B]*n'
-          'B = sp.Matrix.vstack(*L)'
-          'return B' };
+  if n == 0 || m == 0
+    [nrows_A, ncols_A] = size (A);
+    B = zeros (n * nrows_A, m * ncols_A);
```

Comment:
this looks like B would be double (I didn't try it yet!)

but the return type should be sym right?

Reply:
Nice catch! Let's add tests to check this.